exercises/hexadecimal/hexadecimal_test.exs

```diff
@@ -0,0 +1,65 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("hexadecimal.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ChangeTest do
+  use ExUnit.Case
+
+  test "returns 1 when hex is 1" do
+    assert Hexadecimal.to_decimal("1") ==  1
+  end
+
+  @tag :pending
+  test "returns 12 when hex is c" do
+    assert Hexadecimal.to_decimal("c") == 12
+    assert Hexadecimal.to_decimal("C") == 12
```

Comment:
Instead of adding a test for capitalized letters I decided to add assertions to each test that contains letters. What do you think @parkerl and @kotp ?